modules.d/35network-legacy/ifup.sh

```diff
@@ -478,7 +478,11 @@ for p in $(getargs ip=); do
                 ;;
             dhcp6)
                 load_ipv6
-                do_dhcp -6
+                if getargbool 0 rd.net.dhcp6.stateless; then
+                    do_dhcp -6 -S
+                else
+                    do_dhcp -6
+                fi
```

Comment:
There are 2 more calls to `do_dhcp -6`:

https://github.com/dracutdevs/dracut/blob/33a66ed04bdc30eccb172a0cd6dcc36d9d74f825/modules.d/35network-legacy/ifup.sh#L487

https://github.com/dracutdevs/dracut/blob/33a66ed04bdc30eccb172a0cd6dcc36d9d74f825/modules.d/35network-legacy/ifup.sh#L541

Shouldn't the `rd.net.dhcp6.stateless` option also be checked there?